docs/design/architecture.md

```diff
@@ -22,7 +22,12 @@ Quickwit gathers index metadata into a metastore to make them available across t
 
 In a clustered deployment, the metastore is typically a traditional RDBMS like PostgreSQL which we only support today. In a single-server deployment, it’s also possible to rely on a local file or on Amazon S3.
 
-### Distributed message queues
+### Source
```

Comment:
Should we mention push api here?

Reply:
You can mention that it is one special type of source that does not need to be manually configured.